spec/std/crypto/bcrypt_spec.cr

```diff
@@ -8,7 +8,7 @@ describe "Bcrypt" do
     end
   end
 
-  it "raises if cost is to high" do 
+  it "raises if cost is to high" do
     expect_raises ArgumentError, /Invalid cost size/ do
       Crypto::Bcrypt.digest("secret", 32)
     end
```

Comment:
If `digest` fails to raise the error, this spec will take forever to return and report failure. It might be a good idea to add a timeout here (how?).